server/data/educationAndWorkPlanClient.ts

```diff
@@ -30,16 +31,27 @@ export default class EducationAndWorkPlanClient {
     return new RestClient('Education and Work Plan API Client', config.apis.educationAndWorkPlan, token)
   }
 
-  async createGoals(prisonNumber: string, createGoalsRequest: CreateGoalsRequest, token: string): Promise<void> {
-    return EducationAndWorkPlanClient.restClient(token).post({
-      path: `/action-plans/${prisonNumber}/goals`,
-      data: createGoalsRequest,
+  async getActionPlan(prisonNumber: string, token: string): Promise<ActionPlanResponse> {
+    return EducationAndWorkPlanClient.restClient(token).get<ActionPlanResponse>({
+      path: `/action-plans/${prisonNumber}`,
     })
   }
 
-  async getActionPlan(prisonNumber: string, token: string): Promise<ActionPlanResponse> {
-    return EducationAndWorkPlanClient.restClient(token).get<ActionPlanResponse>({
+  async createActionPlan(
```

Comment:
The diff looks bigger than it is because I moved/re-ordered the methods `getActionPlan` and createGoals`; but really what we have here is the introduction of the new `createActionPlan` method